cityofaustin/atd-vz-data
# Add a .nvmrc to the VZ apps

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11023

This PR adds support for [Node Version Manager](https://github.com/nvm-sh/nvm) to the VZV. It looks like @frankhereford already added a .nvmrc file to VZE so thank you 🙏

## Testing
**URL to test:** 
Locally & [netlify](https://deploy-preview-1199--atd-vzv-staging.netlify.app/)

**Steps to test:**
1. To test locally follow steps to set up nvm in the [README](https://github.com/cityofaustin/atd-vz-data/blob/1cde80fe24aff9f3ae2f6923d85f8e89dd75164f/atd-vzv/README.md) of each directory
2. When you add a .nvmrc file to your repository Netlify automatically builds using the version you specified there (pretty cool!). If you check the [build logs](https://app.netlify.com/sites/atd-vzv-staging/deploys/641b46a21022e0000824f424#L54) you can see how it checks .nvmrc and uses v16.

If you want `nvm use` to auto-run whenever you switch directories, I found these [instructions](https://github.com/nvm-sh/nvm#deeper-shell-integration) to set that up

---
#### Ship list
